IFC4x3/PropertyConstants/w/WELDEDANDINSERTABLETOCONTINOUSWELDEDRAIL_3eSc38OQzCyB8vnIaNQTl9.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="WELDEDANDINSERTABLETOCONTINOUSWELDEDRAIL_3eSc38OQzCyB8vnIaNQTl9" Name="WELDEDANDINSERTABLETOCONTINOUSWELDEDRAIL" UniqueId="e87260c8-61af-4cf0-b239-c5291769dbc9" />
```

Comment:
Proposal INSERTABLECONTINOUSWELDEDRAIL.

Reply:
Done, modification to WELDEDANDINSERTABLE. I also add a description to the constant.